vendor/github.com/google/go-containerregistry/pkg/v1/mutate/mutate.go

```diff
@@ -342,6 +342,42 @@ func layerTime(layer v1.Layer, t time.Time) (v1.Layer, error) {
 	return layer, nil
 }
 
+// Squash squashes the image to a single layer
+func Squash(img v1.Image) (v1.Image, error) {
+	newImage := empty.Image
+
+	tar := Extract(img)
+	layer, err := tarball.LayerFromReader(tar)
+	if err != nil {
+		return nil, fmt.Errorf("error squashing layers: %v", err)
```

Comment:
since Go 1.14 is used, I guess it would make sense to use the wrapping helper.

```suggestion
		return nil, fmt.Errorf("error squashing layers: %w", err)
```